src/forms/FormControl/FormControl.stories.tsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react'
+import {ComponentMeta} from '@storybook/react'
+import {FormControl} from '.'
+import {CheckIcon, SearchIcon} from '@primer/octicons-react'
+import {TextInput} from '../TextInput'
+
+export default {
+  title: 'Components/Forms/FormControl',
+  component: FormControl,
+  argTypes: {
+    label: {
+      type: 'string',
+      name: 'label',
+      defaultValue: 'Label of input',
+      description: 'string',
+      table: {
+        category: 'Label'
+      }
+    },
+    leadingText: {
+      type: 'string',
+      name: 'leading text',
+      description: 'string',
+      table: {
+        category: 'Input'
+      }
+    },
+    trailingText: {
+      type: 'string',
+      name: 'trailing text',
+      description: 'string',
+      table: {
+        category: 'Input'
+      }
+    },
+    visuallyHidden: {
+      control: {type: 'boolean'},
+      defaultValue: false,
+      table: {
+        category: 'Label'
+      }
+    },
+    size: {
+      options: [0, 1],
+      mapping: ['medium', 'large'],
+      control: {
+        type: 'inline-radio',
+        labels: ['medium', 'large']
+      },
+      table: {
+        category: 'Input'
+      }
+    },
+    validationStatus: {
+      options: [0, 1, 2], // iterator
+      mapping: [undefined, 'error', 'success'], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['undefined', 'error', 'success']
+      },
+      table: {
+        category: 'Validation'
+      }
+    },
+    validationText: {
+      type: 'string',
+      table: {
+        category: 'Validation'
+      }
+    },
+    fullWidth: {
+      description: 'formerly called Block',
+      control: {type: 'boolean'},
+      defaultValue: false,
+      table: {
+        category: 'Input'
+      }
+    },
+
+    monospace: {
+      description: 'monospace text',
+      control: {type: 'boolean'},
+      defaultValue: false,
+      table: {
+        category: 'Input'
+      }
+    },
+    inset: {
+      description: 'formerly called Contrast',
+      control: {type: 'boolean'},
+      defaultValue: false,
+      table: {
+        category: 'Input'
+      }
```

Comment:
We made the choice to remove inset as there is no guidance about when to use and visually it doesn't add any additional accessibility improvement with contrast.

cc @stvehayes

Reply:
Agreed! Not much reason to keep it in Primer Brand.

Reply:
Removed `inset` in https://github.com/primer/brand/pull/69/commits/4f64c80d2865e3f8ab7902ee25ecff8fa62dbd4f